Sources/App/Models/Package.swift

```diff
@@ -78,7 +78,7 @@ extension Package {
         case analysis
     }
 
-    var isNew: Bool { processingStage != .analysis }
+    var isNew: Bool { status == .new }
```

Comment:
I was done adding the new field and wiring it up when I discovered this in `updatePackage` - where a I was going to reset `isNew`:

```swift
                        if stage == .ingestion && pkg.status == .new {
                            // newly ingested package: leave status == .new for fast-track
                            // analysis
                        } else {
```

🤦 

Still want to double check everything tomorrow but I think this will be all that's needed. Tests all pass, so that's a good sign

Reply:
Thanks, past-Sven! 😅